lua/entities/gmod_wire_expression2/core/core.lua

```diff
@@ -164,20 +169,20 @@ registerOperator("if", "n", "", function(self, args)
 	local op1 = args[3]
 	self.prf = self.prf + args[2]
 
-	local IlikeTrains, result //Yes you do :D
```

Comment:
hah, avo... they once griefed our server, then became friends with the owner.
That's probably how I got "associated" with them.